Source/Stationeers.Addons/Modules/Workshop/WorkshopManagerPatch.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Assets.Scripts;
+using Assets.Scripts.Steam;
+using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Stationeers.Addons.Modules.Workshop
+{
+    public class WorkshopManagerPatch
```

Comment:
As this class contains only static methods, we can safely mark it as static -> `public static class`.